examples/abitest/tests/src/tests.rs

```diff
@@ -26,25 +26,42 @@ const BACKEND_CONFIG_NAME: &str = "backend-config";
 const LOG_CONFIG_NAME: &str = "logging-config";
 const FRONTEND_ENTRYPOINT_NAME: &str = "frontend_oak_main";
 
-// TODO(#541)
-const FRONTEND_WASM: &str = "../../target/wasm32-unknown-unknown/debug/abitest_0_frontend.wasm";
-const BACKEND_WASM: &str = "../../target/wasm32-unknown-unknown/debug/abitest_1_backend.wasm";
+const FRONTEND_MANIFEST: &str = "../module_0/rust/Cargo.toml";
+const BACKEND_MANIFEST: &str = "../module_1/rust/Cargo.toml";
+
+const FRONTEND_WASM_NAME: &str = "abitest_0_frontend.wasm";
+const BACKEND_WASM_NAME: &str = "abitest_1_backend.wasm";
+
+fn build_wasm() -> std::io::Result<Vec<(String, Vec<u8>)>> {
+    let mut frontend = oak_tests::compile_rust_to_wasm(FRONTEND_MANIFEST)?;
```

Comment:
Maybe add an extra `module_name` parameter to `compile_rust_to_wasm` and do the full path creation inside it?  (So this would be `oak_tests::compile_rust_wasm(FRONTEND_MANIFEST, FRONTEND_WASM_NAME)` with no `.push`ing needed)

Reply:
Done